gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/ComposePlugin.kt

```diff
@@ -134,6 +135,21 @@ class ComposePlugin : Plugin<Project> {
                 }
             }
         }
+
+        checkAndWarnAboutUsingJsPlugin(project)
+    }
+
+    private fun checkAndWarnAboutUsingJsPlugin(project: Project) {
+        val msg = "Project has kotlin(\"js\") plugin applied.\n" +
```

Comment:
I think `kotlin(...)` shortcuts are Kotlin DSL only.
`org.jetbrains.kotlin.js` and `org.jetbrains.kotlin.multiplatform` should be used instead.

Also consider using constants instead of literals https://github.com/JetBrains/compose-jb/blob/master/gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/internal/constants.kt#L8

Reply:
fixed